app/javascript/techradar.js

```diff
@@ -20,11 +23,14 @@
             var height = (radius * _.size(this.radar.arcs)) * 2 + (offset * 2);
             var center = {x: width / 2, y: height / 2};
 
+            //noinspection HtmlUnknownTarget
             var linkTemplate = _.template("<a target='_blank' href='${url}'>${title}</a>");
 
+            var extraSpacing = 20; //Extra spacing on the right for texts in tooltip
```

Comment:
Yes this is more a hack than a proper solution - I'd add this to a TODO list somewhere to either
* calculate the lengths of all tooltips, find the furthest one and adjust width based on that
* when repositioning tooltip, calculate if it will be partially out of radar bounds and then change the position accordingly. I think this would not be the best solution UX wise.